packages/stark-ui/src/modules/dropdown/components/_dropdown.component.scss

```diff
@@ -6,33 +6,21 @@
   mat-select {
     margin: 0;
   }
+}
 
-  .mat-form-field {
-    width: 150px;
-  }
-
-  &.stark-full-width {
-    .mat-form-field {
-      width: 100%;
-    }
-  }
+.stark-dropdown-mat-form-field.stark-full-width {
+  width: 100%;
 }
 
 @media #{$tablet-only-query} {
-  .stark-dropdown {
-    .mat-form-field {
-      width: 300px;
-    }
+  .stark-dropdown-mat-form-field {
+    width: 300px;
```

Comment:
I think it's better to use the default material widths for our dropdown. That way if a developer is making a form the alignment should be (more) correct by default. What do you think @christophercr , @SuperITMan ?

Reply:
@SuperITMan with the `mat-form-field-type-mat-select` fix you added these media-queries can be removed without screwing up the ellipsis. 😉